lib/git/lib.rb

```diff
@@ -442,11 +442,7 @@ def config_get(name)
         command('config', ['--get', name])
       end
 
-      if @git_dir
-        Dir.chdir(@git_dir, &do_get)
-      else
-        do_get.call
-      end
+      do_get.call(@git_dir)
```

Comment:
not 💯 this will work, but tests pass. Seems like we might need to pass the `path` to the `config`  command.